languages/ruby/exercises/concept/exceptions/simple_calculator.rb

```diff
@@ -0,0 +1,7 @@
+class SimpleCalculator
+  ALLOWED_OPERATIONS = ['+', '/', '*']
```

Comment:
```suggestion
  ALLOWED_OPERATIONS = ['+', '/', '*'].freeze
```

Reply:
I'm not sure if we've taught `.frozen` yet.

Reply:
We talked about leaving string literal directive and `freeze` for later episodes 😄

Reply:
🤷 sounds fine with me